web/templates/settings/generate_integration_url_modal.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="new-style">
+    <div class="input-group">
+        <label for="integration-name">{{t "Integration"}}</label>
+        <input type="text" id="integration-name" autocomplete="off"/>
+    </div>
+    <div class="input-group">
+        <label for="stream-name">{{t "Stream to use for notifications (optional)"}}</label>
+        <select id="stream-name" class="modal_select update-url">
+            <option value="">{{t "Select stream"}}</option>
+            {{#each streams}}
+                <option>{{this}}</option>
+            {{/each}}
+        </select>
+        <p>{{t "If you do not specify a stream, the bot will send notifications via PMs to the creator of the bot."}}</p>
+    </div>
+    <div class="input-group override-topic-group control-label-disabled">
+        <label class="checkbox">
+            <input
+              type="checkbox"
+              id="override-topic"
+              class="update-url"
```

Comment:
I think we can use a better class here. Probably something like `integration_url_parameter`.

Reply:
Yeah, that class name sounds better.